### PR TITLE
Update Config Editing section

### PR DESCRIPTION
**Pull request by viglesiasce**
Fixes https://github.com/GoogleContainerTools/kpt/issues/2287

### CODE REVIEW

File: site/book/01-getting-started/02-quickstart.md [hunk 1]
```diff
@@ -46,17 +46,26 @@ $ git init; git add .; git commit -m "Pristine nginx package"
...
 ```
 
-Often, you want to automatically mutate and/or validate resources in a package.
-`kpt fn` commands enable you to execute programs called _kpt functions_. For
-instance, you can automatically search and replace all the occurrences of `app`
-name on resources in the package using path expressions:
+### Automating One-time Edits with Functions
+
+The `kpt fn`set of commands enable you to execute programs called _kpt functions_. These
+programs are packaged as Docker containers and take in YAML files, modify or validate them, and then
```
Comment (frankfarzan) [inline site/book/01-getting-started/02-quickstart.md]: Should avoid "Docker" (It's a TM). So just "packaged as containers".

File: site/book/01-getting-started/02-quickstart.md [hunk 2]
```diff
@@ -68,18 +77,30 @@ To see what changes were made to the local package:
...
+  validators:
+    - image: gcr.io/kpt-fn/kubeval:v0.1
+```
+
+You might want to label all resources in the package. To achieve that, you can
+declare `set-labels` function in the `pipeline` section of `Kptfile`. Add this by running the following
+command:
+
+```shell
+cat >> Kptfile <<EOF
```
Comment (frankfarzan) [inline site/book/01-getting-started/02-quickstart.md:98]: Love the HEREDOC for ergonomics

File: site/book/01-getting-started/02-quickstart.md (hunk 1 above)
Comment (frankfarzan) [reply site/book/01-getting-started/02-quickstart.md]: Made the change

**Review by frankfarzan — APPROVED** [review]
Thanks!

**Comment by viglesiasce** [comment]
Thanks for the quick review and merge @frankfarzan!

**Pull request merged**